src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt

```diff
@@ -254,8 +259,10 @@ private fun SpacingContext.blockMustBeMultiLine(): Boolean {
     }
 }
 
+/**
+ * Assumes that left is before right and they are siblings, otherwise bad things will happen.
+ */
 private fun countNonWhitespaceASTNodesBetween(left: ASTNode, right: ASTNode): Int {
-    // TODO(jajakobyly): assert that left is before right and they are siblings
```

Comment:
Uh, forgot about `startOffset` :sweat_smile: 

The problem with assert is that Java's `assert ...` does not evaluate conditions when assertions are disabled, while Kotlin's `assert(...)` do. Just compare these two guys with and without `-ea`:
```java
class Main {
    public static void main(String[] args) {
        // no -ea: prints nothing
        // -ea: prints Mooo
        assert boo();
    }
    static boolean boo() {
        System.out.println("Mooo");
        return false;
    }
}
```
```kotlin
fun main(args: Array<String>) {
    // no -ea: prints Mooo
    // -ea: prints Mooo
    assert(boo())
}

fun boo(): Boolean {
    System.out.println("Mooo")
    return false
}
```